src/BuildPrediction/Predictors/EmbeddedResourceItemsPredictor.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Build.Prediction.Predictors
+{
+    using Microsoft.Build.Evaluation;
+    using Microsoft.Build.Execution;
+
+    /// <summary>
+    /// Finds EmbeddedResource items as inputs.
+    /// </summary>
+    public class EmbeddedResourceItemsPredictor : IProjectPredictor
```

Comment:
>EmbeddedResourceItemsPredictor [](start = 17, length = 30)

Any plan to parse the resx to find more? That's part of ENL related to this path.

Reply:
I didn't see that in ENL, can you shoot me a pointer?

I suspect though that the common case is that those other files are all `<None>` items in the project anyway so that they're actually visible in VS.